content/en/llm_observability/trace_an_llm_application.md

```diff
@@ -22,7 +22,7 @@ If you're new to LLM Observability traces, read the [Core Concepts][3] before pr
 
 ## Instrument your LLM application
 
-<div class="alert alert-info">This guide uses the LLM Observability SDK for Python. If your application is not written in Python, you can complete the steps below with API requests instead of SDK function calls.</a></div>
+<div class="alert alert-info">This guide uses the LLM Observability SDK for Python. If your application is running in a serverless environment, follow the <a href="/llm_observability/sdk/#serverless-setup">serverless setup instructions</a>. If your application is not written in Python, you can complete the steps below with API requests instead of SDK function calls.</div>
```

Comment:
```suggestion
<div class="alert alert-info">This guide uses the LLM Observability SDK for Python. If your application is running in a serverless environment, follow the <a href="/llm_observability/sdk/#aws-lambda-setup">serverless setup instructions</a>. If your application is not written in Python, you can complete the steps below with API requests instead of SDK function calls.</div>
```